todo_app/data/trello_items.py

```diff
@@ -121,16 +130,21 @@ def get_items() -> list:
         card.idBoard = item['idBoard']
         card.description = item['desc']
         card.listName = get_list(card.id)
+        card.taskUrl = f'/tasks/?taskId={card.id}'
```

Comment:
This is a really nice example of a field that could actually be a property - assigned like this it's quite easy for someone to forget to update it if they change a cards id, but if we made it a property then it would be calculated dynamically and always be up to date